src/java.base/share/classes/java/lang/module/ModuleDescriptor.java

```diff
@@ -203,6 +212,21 @@ public Set<Modifier> modifiers() {
             return mods;
         }
 
+        /**
+         * {@return an unmodifiable set of the module {@linkplain AccessFlag
+         * requires flags, possibly empty}}
```

Comment:
Did you mean to include "possibly empty" in the link? Also a minor nit is that the accessFlags methods put "possibly empty" in first sentence of the method description but the other methods have "possibly-empty" in the (more detailed) return description.